api.planx.uk/editor/copyFlow.test.ts

```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
     name: "InsertFlow",
     matchOnVariables: false,
     data: {
-      insert_flows_one: {
+      newFlow: {
```

Comment:
nit: for other `insert_` mutations, it seems you are't prepending `new` - can this just be `flow` for consistency?